### ISSUE TITLE
Clearing Text colors

### ISSUE THREAD

**Issue by Amit2286658** [chinalwb/Android-Rich-text-Editor#89]
How can i clear the text color selection from the colorpicker, for example say,  i select red, and after writing several text, i want to clear the text color so that it returns back to the default text color, i noticed that clicking on the color button does not clear it anyway, rather once selected, there's no way to clear it but i can only toggle between the provided colors,

**Comment by chinalwb** [comment]
You're right. Unlike background color, which you have no choice but in yellow by default, user click it again will clear background color. Text color can only change by selecting one of the colors, by default the text is in black and this color is one of the color options.

Thanks for the interests on ARE.
